lib/pacemaker/pcmk_sched_probes.c

```diff
@@ -508,6 +494,11 @@ order_then_probes(pe_working_set_t * data_set)
 void
 pcmk__order_probes(pe_working_set_t *data_set)
 {
-    order_first_probes(data_set);
+    // Add orderings for "probe then X"
+    for (GList *iter = data_set->resources; iter != NULL; iter = iter->next) {
+        add_restart_orderings_for_rsc((pe_resource_t *) iter->data, data_set);
+    }
```

Comment:
This can be replaced with:
```
g_list_foreach(data_set->resources, (GFunc) add_restart_orderings_for_rsc, data_set);
```

I don't know why I didn't catch this when I made these changes originally.  Perhaps function prototypes have changed, making it possible to pass to g_list_foreach.

Reply:
Added a commit on top for this